### PR TITLE
Documentation

### PR DESCRIPTION
**Pull request by mwuertinger**

### CODE REVIEW

File: demo/two-containers.job.yaml [hunk 1]
```diff
@@ -6,10 +6,10 @@ spec:
   template:
     spec:
       containers:
-      - name: count
+      - name: container1
         image: ubuntu
         command: ["sh", "-c", "for i in `seq 1 10`; do echo $i; sleep 1; done"]
-      - name: count2
+      - name: container2
```
Comment (stephanhorsthemke) [inline demo/two-containers.job.yaml]: could rename the container in 'single-container.yaml as well

File: README.md [hunk 2]
```diff
@@ -1,2 +1,53 @@
...
+Sometimes you just want to run a job in a Kubernetes cluster and wait for its completion while
+keeping an eye on the container output. This usecase frequently appears in deployment pipelines
+eg. for running a smoke test or a database migration before the actual deployment continues. 
+
+## How to install
+```bash
+go get -v -u github.com/egymgmbh/kubejob
+```
+
+## How to use
```
Comment (stephanhorsthemke) [inline README.md]: I think the backoff limit in the job configuration is still quite important correct? Could we maybe add it here or comment it in the .yaml? Or what happens when the backofflimit is greater than 0?

File: demo/two-containers.job.yaml (hunk 1 above)
Comment (mwuertinger) [reply demo/two-containers.job.yaml]: done

File: README.md (hunk 2 above)
Comment (mwuertinger) [reply README.md]: added documentation and a warning if not set correctly

**Pull request merged**